### DISCUSSION TITLE
[HELP] Windows Configuration

### DISCUSSION THREAD

**Discussion by jeeeem** [mfussenegger/nvim-jdtls#364] [Ideas]
Need help with this error
![image](https://user-images.githubusercontent.com/51053482/202712720-242a37c4-65dd-4a11-b099-cae339011438.png)

**Comment by mfussenegger** [comment]
The first error indicates that your `bundles` configuration is wrong. The second and third error indicate that your runtimes configuration is wrong. Either because the path is wrong, it doesn't contain a valid jdk, or because the `name` is invalid. It must be one of the known execution environments to eclipse.jdt.ls. For example `JavaSE-17`.

You'd have to post your configuration
